gslib/__main__.py

```diff
@@ -760,6 +761,18 @@ def _RunNamedCommandAndHandleExceptions(command_runner,
     else:
       _OutputAndExit('You must re-authenticate with your SAML IdP. '
                      'Please run\n$ gsutil config')
+  except exceptions.OAuthError as e:
```

Comment:
Any reason for not combining this with line 706. You can do something like


except (oauth2client.client.HttpAccessTokenRefreshError,  exceptions.OAuthError) as e:
  ...